gender_novels/analysis/dependency_parsing.py

```diff
@@ -0,0 +1,41 @@
+import urllib
+from nltk.parse.stanford import StanfordDependencyParser
+from gender_novels.corpus import Corpus
+from nltk.tokenize import sent_tokenize
+
+if __name__ == "__main__":
+
+    path_to_jar = "assets/stanford-parser.jar"
+    path_to_models_jar = "assets/stanford-parser-3.9.1-models.jar"
+
+    # The jar files are too big to commit directly, so download them
+    url_to_jar = "http://www.trecento.com/dh_lab/nltk_jar/stanford-parser.jar"
+    url_to_models_jar = "http://www.trecento.com/dh_lab/nltk_jar/stanford-parser-3.9.1-models.jar"
+    urllib.request.urlretrieve(url_to_jar, path_to_jar)
+    urllib.request.urlretrieve(url_to_models_jar, path_to_models_jar)
```

Comment:
make sure to check if the file is already there before downloading -- we don't want a 400MB download everytime something is run, just after it's run the first time.